src/components/application_manager/src/resumption/resumption_data_db.cc

```diff
@@ -2222,6 +2309,59 @@ bool ResumptionDataDB::InsertChoiceSetData(
   return true;
 }
 
+bool ResumptionDataDB::InsertUserLocationData(
+    const smart_objects::SmartObject& user_location,
+    int64_t application_primary_key) const {
+  SDL_LOG_AUTO_TRACE();
+  using namespace app_mngr;
+  using namespace smart_objects;
+
+  if (user_location.empty()) {
+    SDL_LOG_INFO("Application doesn't contain user location");
```

Comment:
```suggestion
    SDL_LOG_DEBUG("Application doesn't contain user location");
```

Reply:
fixed in bf0ce24216a70351394e14311bafcfac997606f6